### PR TITLE
Added okShadow field to Person

### PR DESCRIPTION
**Pull request by apluscs**
Also updated all use cases
Next step is to incorporate this with querying scheduledInterviews

### CODE REVIEW

File: src/main/java/com/google/sps/data/PersonRequest.java [hunk 1]
```diff
@@ -64,21 +66,14 @@ public String getLinkedIn() {
...
-            "company",
-            company,
-            "job",
-            job,
-            "linkedIn",
-            linkedIn,
-            "qualifiedJobs")
+            "PutPersonRequest= firstName:%s, lastName:%s, company:%s, job:%s, linkedIn:%s, okShadow: %b, qualifiedJobs: \n",
+            firstName, lastName, company, job, linkedIn, okShadow)
         + qualifiedJobsToString();
```
Comment (meagancheese) [inline src/main/java/com/google/sps/data/PersonRequest.java]: This is weird, is there a reason qualifiedJobsToString() can't be within String.format?

Comment (meagancheese) [reply src/main/java/com/google/sps/data/PersonRequest.java]: Also, jobs should be before shadow to be consistent throughout.

**Review by meagancheese — CHANGES_REQUESTED** [review]
One main thing, then make sure to resolve conflicts with master.

File: src/main/java/com/google/sps/data/PersonRequest.java (hunk 1 above)
Comment (apluscs) [reply src/main/java/com/google/sps/data/PersonRequest.java]: Just realized qualifiedJobs is printed nicely, you were right.

**Review by meagancheese — APPROVED** [review]

**Review by GrantComm — APPROVED** [review]

**Review by jalexanderqed — APPROVED** [review]

**Pull request merged**